src/video/wayland/SDL_waylandwindow.h

```diff
@@ -63,6 +63,12 @@ typedef struct {
     SDL_Window *sdlwindow;
     SDL_VideoData *waylandData;
     struct wl_surface *surface;
+    struct wl_surface *mask_surface;
+    struct wl_surface *input_surface;
+    struct wl_buffer *mask_surface_buffer;
+    struct wl_buffer *input_surface_buffer;
+    struct wl_subsurface *mask_subsurface;
+    struct wl_subsurface *draw_subsurface;
```

Comment:
Documenting these variables as being part of mode emulation may help with readability - most of these variables will be used at all times but these might not.